packages/astro/CHANGELOG.md

```diff
@@ -20,7 +20,7 @@
 
   Astro's static analysis to determine which `<script>` tags to bundle together just got a little smarter!
 
-  Astro create bundles that optimize script usage between pages and place them in the head of the document so that they are downloaded as early as possible. One limitation to Astro's existing approach has been that you could not dynamically use hoisted scripts. Each page received the same, all-inclusive bundle whether or not every script was needed on that page.
+  Astro create bundles that optimize script usage between pages and place them in the head of the document so that they are downloaded as early as possible. Previously, if a component appeared anywhere in a page's import tree, its script would be included in the page's bundle - even if the component went unused in the page!
```

Comment:
```suggestion
  Astro create bundles that optimize script usage between pages and place them in the head of the document so that they are downloaded as early as possible. Previously, if a component appeared anywhere in a page's import tree, its script would be included in the page's bundle, even if the component went unused in the page.
```